packages/notifications/src/elements/global-alert/GlobalAlert.tsx

```diff
@@ -38,7 +38,7 @@ const GlobalAlertComponent = forwardRef<HTMLDivElement, IGlobalAlertProps>(
 
     return (
       /* [1] */
-      <ThemeProvider theme={theme => ({ ...theme, colors: { ...theme.colors, base: 'light' } })}>
+      <ThemeProvider theme={theme => ({ ...theme!, colors: { ...theme!.colors, base: 'light' } })}>
```

Comment:
```suggestion
      <ThemeProvider theme={theme => ({ ...theme, colors: { ...theme.colors, base: 'light' } })}>
```

Change the `ThemeProvider` import to `@zendeskgarden/react-theming`.